packages/common/feed-store/src/feed-store.ts

```diff
@@ -68,7 +68,7 @@ export class FeedStore<T extends {}> {
       }
     }
 
-    const core = this._factory.createFeed(feedKey, { writable });
+    const core = this._factory.createFeed(feedKey, { writable, sparse });
```

Comment:
check if sparse option matches that of an already open feed (see `writable` above)